src/views/studio/modals/transfer-host-tile.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+import classNames from 'classnames';
+
+const TransferHostTile = ({
+    className, username, image, isCreator, selected, handleSelected
+}) => (
+    <div
+        className={classNames('studio-member-tile', className, {
+            'transfer-host-tile-selected': selected
+        })}
+        onClick={handleSelected}
+    >
+        <img
+            className="studio-member-image"
+            src={image}
+        />
+        <div className="studio-member-info">
+            <div
+                className={classNames('studio-member-name',
+                    {'transfer-host-name-selected': selected}
+                )}
+
+            >
+                {username}
+            </div>
+            {isCreator && <div className="studio-member-role"><FormattedMessage id="studio.HostRole" /></div>}
```

Comment:
We might not need this anymore (or the isCreator prop)

Reply:
all set